cloudsync/tasks.py

```diff
@@ -1,16 +1,14 @@
 """
 Tasks for cloudsync app
 """
-
 import os
 import re
-from urllib.parse import unquote
-
-import boto3
+import urllib
```

Comment:
Thanks, I misunderstood the earlier comments to mean putting all 'import' statements before 'from' statements regardless of package source, I'll update this.